src/apis/Notification.ts

```diff
@@ -0,0 +1,14 @@
+import instance from './axios';
+
+export const getNotification = async (token: string) => {
+  try {
+    const { data } = await instance.get('/notifications', {
+      headers: { Authorization: `Bearer ${token}` },
+    });
+    console.log(data);
+
+    return data;
+  } catch (error) {
+    console.error(error);
```

Comment:
error처리는 여기서 axios 에러만 처리해주는게 좋다고 판단되서 고칠 수 있으면 고쳐주시면 감사하겠습니다!!

```suggestion
  } catch (error) {
  if(error instanceof AxiosError){
    console.error(error.message);
  }
```

이런 느낌이에요!!

Reply:
동의합니다!!